README.md

```
@@ -29,21 +75,210 @@ To use an example:
+See [LICENSE](LICENSE) for details.
 
 ## Status
 
-🚧 V0.1.0 - Work in progress
+🚧 **V0.1.0** - Work in progress
+
+The application is actively being developed. See [docs/specifications/PROGRESS.md](docs/specifications/PROGRESS.md) for implementation status.
+
+## Links
+
+- **Repository**: https://github.com/ttu/emergency-supply-tracker
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)